src/frontend/staff-portal/src/app/services/dispute.service.ts

```diff
@@ -470,9 +470,9 @@ export class DisputeService implements IDisputeService {
     // split up where commas occur and stuff in address lines 1,2,3
     if (Dispute.address) {
       let addressLines = Dispute.address.split(",");
-      if (addressLines.length > 0) dispute.addressLine1 = addressLines[0].length > 100 ? addressLines[0].substring(0,100) : addressLines[0];
-      if (addressLines.length > 1) dispute.addressLine2 = addressLines[1].length > 100 ? addressLines[1].substring(0,100) : addressLines[1];
-      if (addressLines.length > 2) dispute.addressLine3 = addressLines[2].length > 100 ? addressLines[2].substring(0,100) : addressLines[2];
+      if (addressLines.length > 0) dispute.addressLine1 = addressLines[0].length > 100 ? addressLines[0].substring(0, 100) : addressLines[0];
```

Comment:
Why do we do the 'addressLines[0].length > 100' check? if you just want the first 100 characters, just call the substring. If the string is less than 100, then the entire string is retunred.